### PR TITLE
[doc] Added link to contributor's guide

### PR DESCRIPTION
**Pull request by eldernewborn**
## Summary, imperative, start upper case, don't end with a period
added link to contributors guide to README.md

## How was this PR tested?
N/A
## Does this PR introduce any user-facing changes?
- [X] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -145,10 +145,11 @@ Feel free to engage with the community using our:
   - Archived and publicly searchable on [Linen](http://linen.venicedb.org)
 - [LinkedIn group](https://www.linkedin.com/groups/14129519/)
 - [GitHub issues](https://github.com/linkedin/venice/issues)
-- [Contributor's guide](dev_guide/how_to/contribution_agreement)
+- [Contributor's guide](https://venicedb.org/docs/dev_guide/how_to)
+- [Contributor's agreement](dev_guide/how_to/contribution_agreement)
```
Comment (nisargthakkar) [inline docs/README.md:149]: Can you please also fix this path? https://venicedb.org/docs/dev_guide/how_to/contribution_agreement

**Review by nisargthakkar — COMMENTED** [review]
Thanks for the PR. Way to go 🚀 

Can you please also update the PR title to follow the standards? See Point 6 on this doc: https://venicedb.org/docs/dev_guide/how_to/recommended_development_workflow#pull-request

File: docs/README.md (hunk 1 above)
Comment (FelixGV) [reply docs/README.md:149]: Hmm... so the README page can resolve relative links when seen from venicedb.org but not when seen from the rendering on GitHub? Is that the issue?

Comment (nisargthakkar) [reply docs/README.md]: I verified that we can preserve links in README and the doc website by linking to the `.md` file.

Hey @eldernewborn. Can you please make these changes to your PR?
```
- [Contributor's guide](./dev_guide/how_to/how_to.md)
- [Contributor's agreement](./dev_guide/how_to/CONTRIBUTING.md)
```

I fixed all broken doc links that I will merge after your PR is done: https://github.com/linkedin/venice/pull/961/files

**Review by nisargthakkar — APPROVED** [review]

**Pull request merged**